tethys_apps/cli/gen_templates/settings

```diff
@@ -83,6 +83,33 @@ ALLOWED_HOSTS = [{% if allowed_host %}'{{ allowed_host }}'{% endif %}]
 
 # Application definition
 
+# Placeholders for the ID's required by various web-analytics services supported by Django-Analytical.
```

Comment:
@rileyhales move this block of settings down to the bottom of the settings template.